pygridgain/connection/__init__.py

```diff
@@ -318,6 +318,8 @@ def _connect_version(
                     client_patch=protocol_version[2],
                     **hs_response
                 )
+            elif error_text.find('The user name or password is incorrect') != -1:
```

Comment:
Let's check for `ClientStatus.AUTH_FAILED = 2000` instead of parsing the error text

Reply:
You are right, there is a field since 1.1.0 I've completely forgot of. Will fix.